istio.deps

```diff
@@ -4,7 +4,7 @@
 		"name": "PROXY_REPO_SHA",
 		"repoName": "proxy",
 		"file": "",
-		"lastStableSHA": "8ddb2aab27a5d311adf81995f998df62e4c0e885"
+		"lastStableSHA": "a5d5a464251e940f2d59a23ec598deb0277a8dd3"
```

Comment:
Matches the sha in https://github.com/duderino/proxy/blob/0c32f7a781eeeab893cf214d056bbc91883e0eb7/istio.deps